src/dsp_tools/resources/schema/project.json

```diff
@@ -16,18 +16,15 @@
         },
         "ncname": {
             "type": "string",
-            "pattern": "^[a-zA-Z_][\\w.-]*$",
-            "additionalProperties": false
```

Comment:
Here I'm unsure if "additionalProperties" should be kept or removed. (And also in the similar cases below).